<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="9">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="loadFolderButton">
             <property name="text">
              <string>Load Folder</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="loadFileButton">
             <property name="text">
              <string>Load Music File</string>
             </property>
             <property name="icon">
              <iconset resource="appicons.qrc">
               <normaloff>:/png/four30.png</normaloff>:/png/four30.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="QListWidget" name="mp3List"/>
         </item>
        </layout>
       </item>
       <item row="3" column="6">
        <widget class="QPushButton" name="next">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="appicons.qrc">
           <normaloff>:/png/next10.png</normaloff>:/png/next10.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="previous">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="appicons.qrc">
           <normaloff>:/png/back12.png</normaloff>:/png/back12.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QPushButton" name="pauseButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="appicons.qrc">
           <normaloff>:/png/pause15.png</normaloff>:/png/pause15.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Volume</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSlider" name="volumeSlider">
         <property name="value">
          <number>80</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="appicons.qrc">
           <normaloff>:/png/square64.png</normaloff>:/png/square64.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="playButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="appicons.qrc">
           <normaloff>:/png/play43.png</normaloff>:/png/play43.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="7">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QSlider" name="position">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="appicons.qrc"/>
 </resources>
 <connections/>
</ui>
